content/en/blog/2024/otel-generative-ai/index.md

```diff
@@ -0,0 +1,163 @@
+---
+title: OpenTelemetry for Generative AI
+linkTitle: OpenTelemetry for Generative AI
+date: 2024-11-09
+author: >-
+  [Drew Robbins](https://github.com/drewby) (Microsoft),  [Liudmila
+  Molkova](https://github.com/lmolkova) (Microsoft)
+issue: [#5581](https://github.com/open-telemetry/opentelemetry.io/issues/5581)
+sig: SIG GenAI Observability
+---
+
+As organizations increasingly adopt Large Language Models (LLMs) and other
+generative AI technologies, ensuring reliable performance, efficiency, and
+safety is essential to meet user expectations, optimize resource costs, and
+safeguard against unintended outputs. Effective observability for AI operations,
+behaviors, and outcomes can help meet these goals. OpenTelemetry is being
+enhanced to support these needs specifically for generative AI.
+
+Two primary assets are in development to make this possible: **Semantic
+Conventions** and an **Instrumentation Library**.
+
+[**Semantic Conventions**](https://opentelemetry.io/docs/concepts/semantic-conventions/)
+establish standardized guidelines for how telemetry data is structured and
+collected across platforms, defining inputs, outputs, and operational details.
+For generative AI, these conventions streamline monitoring, troubleshooting, and
+optimizing AI models by standardizing attributes such as model parameters,
+response metadata, and token usage. This consistency supports better
+observability across tools, environments, and APIs, helping organizations track
+performance, cost, and safety with ease.
+
+The
+[**Instrumentation Library**](https://opentelemetry.io/docs/specs/otel/overview/#instrumentation-libraries)
+is being developed within the
+[OpenTelemetry Python Contrib](https://github.com/open-telemetry/opentelemetry-python-contrib)
```

Comment:
how about link to https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation-genai?

Reply:
We have the link later in the doc when we talk about the specifics of the library.

Reply:
Yes, Line 94 is a link for a specified link for openai instrumentation, but here should be the folder link of `instrumentation-genai`?

```suggestion
[OpenTelemetry Python Contrib](https://github.com/open-telemetry/opentelemetry-python-contrib) under `[instrumentation-genai](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation-genai)`
```